UI/window-basic-settings.cpp

```diff
@@ -2061,7 +2061,7 @@ static inline QString makeFormatToolTip()
 #define SRT_PROTOCOL "srt"
 #define RIST_PROTOCOL "rist"
 
-inline bool allowsMultiTrack(const char *protocol)
+inline bool OBSBasicSettings::allowsMultiTrack(const char *protocol)
```

Comment:
The inline keyword is now in the declaration in the header file.

```suggestion
bool OBSBasicSettings::allowsMultiTrack(const char *protocol)
```

Reply:
done